ote/src/cljs/ote/views/service_viewer.cljs

```diff
@@ -493,32 +531,362 @@
              [common-ui/information-row-with-option (tr [:common-texts :ending-time]) end-date true]]]))]
       [information-row-with-selection
        (tr [:field-labels :transport-service-common ::t-service/service-hours-info])
-       (format-descriptions info) true]])
+       (format-descriptions info) true]]
+     [spacer]]))
+
+(defn- vehicle-and-price [title vehicle-classes vehicle-price-url]
+  [:section
+   [:h4 title]
+   (let [vehicle-classes vehicle-classes]
+     [:div
+      [common-ui/information-row-with-option (tr [:service-viewer :url]) vehicle-price-url true]
+      (doall
+        (for [vc vehicle-classes
+              :let [license-required (get vc ::t-service/license-required)
+                    vehicle-type (get vc ::t-service/vehicle-type)
+                    minimum-age (get vc ::t-service/minimum-age)]]
+          ^{:key (str vc)}
+          [info-sections-2-cols vehicle-type
+           [:div
+            [common-ui/information-row-with-option (tr [:field-labels :rentals :ote.db.transport-service/license-required]) license-required false]
+            (doall
+              (for [price-row (::t-service/price-classes vc)
+                    :let [price (::t-service/price-per-unit price-row)]]
+                ^{:key (str vehicle-type "-" license-required "-" price)}
+                [common-ui/information-row-with-option (tr [:field-labels :rentals :ote.db.transport-service/price-classes]) (str price " €") false]))]
+           [:div
+            [common-ui/information-row-with-option (tr [:field-labels :rentals :ote.db.transport-service/minimum-age]) minimum-age false]
+            (doall
+              (for [price-row (::t-service/price-classes vc)
+                    :let [unit (::t-service/unit price-row)]]
+                ^{:key (str vehicle-type "-" minimum-age "-" unit)}
+                [common-ui/information-row-with-option (tr [:service-viewer :pricing-basis]) unit false]))]]))])
+   [spacer]])
```

Comment:
If the spacer is not involved to this sub-component, but meant to separate two different sub-components, then spacer should be moved one level up outside this sub-component into the `case`.
Reasoning: if this is re-used, a spacer is not necessarily appropriate there.

Reply:
Spacer was used this way in all sub-components in the original design.
I decided not to refactor this design in this pr. (I ended up to fix other pages which is not wise)